README.md

```diff
@@ -26,10 +25,8 @@ Kubernetes uses etcd as its datastore. The official infrastructure requirements
 ## Deploying CFCR
 
 1. Upload the [latest Xenial stemcell](https://bosh.io/stemcells/#ubuntu-xenial) to the director.
-1. Upload the latest kubo-release to the director.
-    ```
-    bosh upload-release https://bosh.io/d/github.com/cloudfoundry-incubator/kubo-release
-    ```
+
+1. Untar the kubo-deployment tarball and rename it `kubo-deployment`
```

Comment:
Can we move away from suggesting to use the tarball. We have master which should always be stable for people to use.